src/components/Task/Drawer/Form/Form.stories.tsx

```diff
@@ -103,7 +105,9 @@ export const Persisted = (): ReactElement => {
     <MockedProvider
       mocks={[
         getDataForTaskDrawerMock(),
+        getTasksForTaskListMock(),
         { ...updateTaskMutationMock(), delay: 500 },
+        { ...deleteTaskMutationMock(), delay: 1000 },
```

Comment:
@OzzieOrca So the mutation works in this storybook, but I ran into an error as it's trying to update the cache after the mutation. Mainly because the cache does not have the data from the ```getTasksForTaskListQuery```. I can add the mock here, but since it doesn't get called in the file, it's not in the cache.
<img width="359" alt="Screen Shot 2021-03-25 at 10 14 10 AM" src="https://user-images.githubusercontent.com/39680460/112487745-59016e00-8d53-11eb-9021-4123b377b149.png">

Any ideas on how I could, I guess, force it to run the query but just in this storybook? Or can I have some fallback logic in the update function to query for that data if there is nothing in the cache?

Reply:
Actually, I figured it out! 🙂